CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # DocumenterTools.jl changelog
 
+## Version `v0.1.18`
+
+* ![Enhancement][badge-enhancement] Allow OpenSSH_jll 9 as dependency. ([#74][github-74])
```

Comment:
Also needs a link reference down at the bottom.

Reply:
Done.